### PR TITLE
Web UI updates in subtitle and broadcast texts.

### PR DESCRIPTION
**Pull request by piotr-travisci**
Texts updates according
https://github.com/travis-ci/bugs-questions-comments-ideas/issues/106

### CODE REVIEW

**Comment by piotr-travisci** [comment]
Thanks for suggestion. I have changed indentation of in my ide, so my pull request looks better. It's more readable ;-)

File: app/templates/settings.hbs [hunk 1]
```diff
@@ -1,139 +1,153 @@
...
-
-  </section>
-{{/if}}
+	<section class="settings-section">
+		<h2 class="small-title">General</h2>
+		<ul class="settings-list--columns">
+			<li>
+				{{settings-switch active=model.settings.build_pushes repo=repo description="Build pushed branches" key="build_pushes"}}
+				<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-branches" title="about branch updates"
+					class="settings-tooltip">
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: For consistency, please either put all attributes in one line or put each one on its own line (when it's too long), like this:
```
<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-branches" 
  title="about branch updates"
	class="settings-tooltip"
>
```
When multi-line, please put closing bracket to the new line too.

File: app/templates/settings.hbs [hunk 2]
```diff
@@ -1,139 +1,153 @@
...
+					{{ember-tooltip text='Read more about branch updates'}}
+					{{svg-jar 'icon-help' class="icon-help"}}</a>
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Closing tag should be on the new line, for better readability - to easier find closing tags when scanning code.

File: app/templates/settings.hbs [hunk 3]
```diff
@@ -1,139 +1,153 @@
...
+				<a href="https://docs.travis-ci.com/user/web-ui/#build-pushed-pull-requests" title="about pull request updates"
+					class="settings-tooltip">
+					{{ember-tooltip text='Read more about pull request updates'}}
+					{{svg-jar 'icon-help' class="icon-help"}}</a>
+			</li>
+		</ul>
+	</section>
+
+	{{#if showAutoCancellationSwitches}}
+	<section class="settings-section auto-cancellation">
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: This is a nested block inside `{{#if}}` so it should be indented

File: app/templates/settings.hbs [hunk 4]
```diff
@@ -1,139 +1,153 @@
...
+		<h2 class="small-title display-inline">Auto Cancellation</h2>
+
+		<p>Auto Cancellation allows you to only run builds for the latest commits in the queue. This setting can be
+			applied to builds for Branch builds and Pull Request builds separately. Builds will only be canceled if they
+			are waiting to run, allowing for any running jobs to finish.</p>
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: It's be better to put all the text on new lines, indented inside `p`:
```
<p>
  Auto Cancellation allows you to only run builds for the latest commits in the queue. This setting can be
	applied to builds for Branch builds and Pull Request builds separately. Builds will only be canceled if they
	are waiting to run, allowing for any running jobs to finish.
</p>
```

File: app/templates/settings.hbs [hunk 5]
```diff
@@ -1,139 +1,153 @@
...
+			</li>
+			<li>
+				{{settings-switch active=model.settings.auto_cancel_pull_requests repo=repo description="Auto cancel pull request builds" key="auto_cancel_pull_requests"}}
+			</li>
+		</ul>
+	</section>
+	{{/if}}
+
+	{{#if showAllowConfigImportsSwitch}}
+	<section class="settings-section config-import">
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Nested block should be indented.

File: app/templates/settings.hbs [hunk 6]
```diff
@@ -1,139 +1,153 @@
...
+	{{/if}}
+
+	<section class="settings-section">
+		<h2 class="small-title">Environment Variables</h2>
+
+		<p>Customize your build using environment variables. For secure tips on generating private keys
+			<a
+				href='https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs'>
+				read our documentation
+			</a>.</p>
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: `a` looks like a one-liner as it has only one attribute, and they all should be nested inside `p`:
```
<p>
  Customize your build using environment variables. For secure tips on generating private keys
  <a href='https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs'>
    read our documentation
  </a>.
</p>
```

File: app/templates/settings.hbs [hunk 7]
```diff
@@ -1,139 +1,153 @@
...
+
+		<ul class="settings-list--envvars">
+			{{#each envVars as |envVar|}}
+			{{env-var envVar=envVar}}
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Nested block should be indented

File: app/templates/settings.hbs [hunk 8]
```diff
@@ -1,139 +1,153 @@
...
+			{{/each}}
+		</ul>
+
+		<div id='env-var-secrecy'>
+			<p>
+				{{svg-jar 'icon-megaphone' class='icon icon-megaphone'}}
+				<span class="label-align">
+					If your secret variable has especial characters like &, escape it by adding \ in front of it. For
+					example ma&w!doc would be typed as ma\&w\!doc.
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: I would suggest to use HTML Entities for `&`: `&amp;`

File: app/templates/settings.hbs [hunk 9]
```diff
@@ -1,139 +1,153 @@
...
+		{{add-env-var repo=repo}}
+
+	</section>
+
+	<section class="settings-section">
+		<h2 class="small-title display-inline">Cron Jobs</h2>
+
+		<ul class="settings-list--crons">
+			{{#each cronJobs as |cron|}}
+			{{cron-job cron=cron}}
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Nested block should be indented

File: app/templates/settings.hbs [hunk 10]
```diff
@@ -1,139 +1,153 @@
...
+			{{/each}}
+		</ul>
+
+		{{#if sortedBranchesWithoutCron.length}}
+		{{add-cron-job branches=sortedBranchesWithoutCron}}
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Nested block should be indented

File: app/templates/settings.hbs [hunk 11]
```diff
@@ -1,139 +1,153 @@
...
+		{{else}}
+		<p class="text-medium">There are no branches you can currently add crons to.</p>
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: Nested block should be indented

**Comment by piotr-travisci** [comment]
@artursmirnov you are right.
Is it looking good now?
styling

**Comment by artursmirnov** [comment]
Hey @piotr-travisci, the link looks good to me, but for `code` segments I was thinking about something more familiar to our users, something like inline code segments look on GitHub and Slack (example: `code_segment` - monospace font, background, maybe border). We have similar styles already, but only for `notice-banner`s: https://github.com/travis-ci/travis-web/blob/master/app/styles/app/modules/notice.scss#L54-L59
It would be nice to adopt these styles.
And this is an example of their use: https://github.com/travis-ci/travis-web/blob/aab8e47ae7aa5710e04e9235e9e97f2b36de7ab0/app/templates/components/job-infrastructure-notification.hbs

**Comment by piotr-travisci** [comment]
Hey @artursmirnov,

Updated styling for code snippets
code_snippets_styling

File: app/templates/settings.hbs [hunk 12]
```diff
@@ -45,7 +45,12 @@
   <section class="settings-section">
     <h2 class="small-title">Environment Variables</h2>
 
-    <p>Notice that the values are not escaped when your builds are executed. Special characters (for bash) should be escaped accordingly.</p>
+    <p>
+      Customize your build using environment variables. For secure tips on generating private keys
+      <a href="https://docs.travis-ci.com/user/best-practices-security#recommendations-on-how-to-avoid-leaking-secrets-to-build-logs">
```
Comment (artursmirnov) [inline app/templates/settings.hbs]: This link should open in a new tab. Anyways, there is an [`external-link-to`](https://github.com/travis-ci/travis-web/blob/master/app/components/external-link-to.js) component that should be used for external links. It internally bounds all necessary attributes and takes care of some security issues. 

Also we strive to keep all our external URLs in config section: https://github.com/travis-ci/travis-web/blob/master/config/environment.js#L39-L62 - you can add the URL there and then use it here with a `config-get` helper:

```
{{#external-link-to href=(config-get 'urls.urlKey')}}
  read our documentation
{{/external-link-to}}
```

**Pull request merged**